packages/maskbook/src/plugins/Snapshot/hooks/useProposalIdentifier.ts

```diff
@@ -0,0 +1,10 @@
+import type { ProposalIdentifier } from '../types'
+
+export function useProposalIdentifier(url: string): ProposalIdentifier {
```

Comment:
This is not a hooks (not using any React features) no meaning to limit it to React